packages/app-desktop/gui/NoteEditor/StatusBar.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import ToolbarButton from '../ToolbarButton/ToolbarButton';
+import { ToolbarButtonInfo } from '@joplin/lib/services/commands/ToolbarButtonUtils';
+import CommandService from '@joplin/lib/services/CommandService';
+import { themeStyle } from '@joplin/lib/theme';
+import { AppState } from '../../app.reducer';
+import { connect } from 'react-redux';
+import { TagEntity } from '@joplin/lib/services/database/types';
+import TagList from '../TagList';
+import { _ } from '@joplin/lib/locale';
+import { useCallback } from 'react';
+import KeymapService from '@joplin/lib/services/KeymapService';
+
+interface Props {
+	themeId: number;
+	tabMovesFocus: boolean;
+	noteId: string;
+	setTagsToolbarButtonInfo: ToolbarButtonInfo;
+	selectedNoteTags: TagEntity[];
+}
+
+interface StatusIndicatorProps {
+	commandName: string;
+	showWhenUnfocused: boolean;
+	// Even if not visible, [label] should reflect the current state
+	// of the indicator.
+	label: string;
```

Comment:
To better support screen readers and keyboard-only users:
- The status indicator announces changes to the setting with [aria-live=polite](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live).
   - As a result, pressing <kbd>ctrl</kbd>-<kbd>m</kbd> to disable keyboard focus announces "tab indents" (see [SC 4.1.3. "Status Messages"](https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html).
- Changing the keyboard focus mode (e.g. by pressing enter) when the status indicator has focus keeps the status indicator visible but changes its label to "Tab indents".